seqr/utils/elasticsearch/es_search.py

```diff
@@ -63,12 +65,12 @@ def __init__(self, families, previous_search_results=None, skip_unaffected_famil
             if len(self.samples_by_family_index) < 1:
                 raise Exception('Inheritance based search is disabled in families with no affected individuals')
 
-        self._indices = self.samples_by_family_index.keys()
+        self._indices = sorted(list(self.samples_by_family_index.keys()), reverse = True)
```

Comment:
It is weird that the behavior of the code depends on the order of `self._indices` very much. Without sorting, the results of `elasticsearch` keep changing between each run.